examples/basics/visuals/volume_plane.py

```diff
@@ -0,0 +1,105 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# Copyright (c) Vispy Development Team. All Rights Reserved.
+# Distributed under the (new) BSD License. See LICENSE.txt for more info.
+# -----------------------------------------------------------------------------
+# vispy: gallery 2
+"""
+Example volume rendering with plane rendering
```

Comment:
Would you mind adding a `===` line underneath this and moving this example to `examples/scene`? I also assume this example starts by showing a plane rather than the whole volume (the gallery would capture the whole volume for the website which isn't as "fun").

Github is also showing that you did "something" with the doc submodule. Did you make changes to any files? Maybe you need to update the `doc` submodule:

```
cd doc
git pull  # assuming on upstream main
cd ..
git add doc
git commit
```

Reply:
good catch! Not sure why but that *something* always happens with my local setup in pycharm... should probably read into submodules a bit - will fix!

Reply:
and I'll make sure the init plane is an interesting one for the gallery :)